### PR TITLE
Document business connections

### PR DESCRIPTION
**Pull request by KnorpelSenf**
https://core.telegram.org/bots#manage-your-business

### CODE REVIEW

**Review by kolay-v — CHANGES_REQUESTED** [review]

File: site/docs/advanced/business.md [hunk 1]
```diff
@@ -1,13 +1,138 @@
...
-- what is it
-- basic concepts and short code snippets
-  - chats
-  - users
-  - connection identifiers
-- more support by grammY
-  - filter queries
-- tips and tricks
-  - database
+Telegram Business allows your private chat with another (human) user to be managed by a bot.
```
[the remark this answers is not in the document]
Comment (KnorpelSenf) [reply site/docs/advanced/business.md:3]: Yes, later

**Comment by kolay-v** [comment]
everything else is looking great

File: site/docs/advanced/business.md [hunk 2]
```diff
@@ -1,13 +1,138 @@
...
+    // Automatically respond to all customer questions.
+    if (ctx.msg.text.endsWith("?")) {
+      await ctx.reply("Soon.");
+    }
+  }
+);
+```
+
+This will look as if you had sent the message yourself.
+There is no way to tell for you or your customer whether the message was sent manually or via your bot.
```
[the remark this answers is not in the document]
Comment (KnorpelSenf) [reply site/docs/advanced/business.md:73]: Like this?
```suggestion
There is no way to tell for your customer whether the message was sent manually or via your bot.
(You will see a small indicator for this, though.)
```

File: site/docs/advanced/business.md [hunk 3]
```diff
@@ -1,13 +1,138 @@
...
+(But your bot is probably much faster at replying than you.
+Sorry.)
+
+## Going Further
+
+There are a few more things to consider when integrating your bot with Telegram Business.
+We will cover a few aspects here briefly.
+
+### Editing or Deleting Business Messages
```
[the remark this answers is not in the document]
Comment (KnorpelSenf) [reply site/docs/advanced/business.md:82]: I believe that people will ask “how do I edit or a delete a message in the managed chat” so I want to talk explicitly about editing or deleting messages (and say that it's impossible).

File: site/docs/advanced/business.md [hunk 4]
```diff
@@ -1,13 +1,138 @@
...
+### Receiving Business Messages
+
+Once a business connection was set up, the bot will **receive messages** from _both chat participants_.
+
+```ts
+bot.on("business_message", async (ctx) => {
+  // Access the message object.
+  const message = ctx.businessMessage;
+  // Shortcuts work as expected.
+  const m = ctx.msg;
```
[the remark this answers is not in the document]
Comment (kolay-v) [reply site/docs/advanced/business.md:24]: maybe it is fine. but i would prefer making same as other docs examples.

[the remark this answers is not in the document]
Comment (kolay-v) [reply site/docs/advanced/business.md:24]: okay

File: site/docs/advanced/business.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (kolay-v) [reply site/docs/advanced/business.md:82]: okay

File: site/docs/advanced/business.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (KnorpelSenf) [reply site/docs/advanced/business.md]: done

**Review by kolay-v — APPROVED** [review]

File: site/docs/advanced/business.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
+# Telegram Business
+
+Telegram Business allows your private chat with another (human) user to be managed by a bot
```
Comment (roj1512) [inline site/docs/advanced/business.md:3]: Punc

File: site/docs/advanced/business.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+This includes sending and receiving messages on your behalf.
+Typically, this is useful if you run your business on Telegram, and that other user is a customer.
```
Comment (roj1512) [inline site/docs/advanced/business.md:5]:
```suggestion
Typically, this is useful if you run your business on Telegram, and that other users are customers.
```

File: site/docs/advanced/business.md [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+
+> If you are not familiar with Telegram Business yet, check out the [official docs](https://core.telegram.org/bots#manage-your-business) by Telegram before you continue.
+
+Naturally, grammY has full support for this.
+
+## Handling Business Messages
+
+A bot can manage a private chat between two users via Telegram Business.
```
Comment (roj1512) [inline site/docs/advanced/business.md:13]: Optional suggestion. Resolve without commenting.

```suggestion
A bot can manage a private chat between two users via Telegram Business—an account that is subscribed to Telegram's business subscription.
```

File: site/docs/advanced/business.md (hunk 6 above)
Comment (KnorpelSenf) [reply site/docs/advanced/business.md:5]: We talk about the chat between the reader and a different user. This sentence refers to that other user in said chat, not to any number of people.

**Review by roj1512 — APPROVED** [review]

File: site/docs/uk/advanced/business.md [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+);
+```
+
+Це виглядатиме так, ніби ви відправили повідомлення особисто.
+Ваш клієнт не зможе визначити, чи було повідомлення надіслано вручну, чи через бота.
+Однак ви побачите невеликий індикатор щодо цього.
+Але ваш бот, ймовірно, відповідає набагато швидше, ніж ви.
+Перепрошуємо.
+
+## Просування
```
Comment (niusia-ua) [inline site/docs/uk/advanced/business.md:79]: I can't find a suitable translation for "Going further". @deptyped @LWJerri, if you have any ideas, please leave them in this comment

Comment (KnorpelSenf) [reply site/docs/uk/advanced/business.md:79]: Just translate something else. Here are a few options:
- Next steps
- Advanced features
- How to continue
- Going beyond the basics
- Whatever floats your goat

File: site/docs/id/advanced/business.md [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+Meski demikian, bot kamu **TIDAK** bisa mengedit atau menghapus pesan di chat tersebut.
+Bot juga **TIDAK** bisa meneruskan pesan dari chat tersebut ataupun menyalinnya ke tempat lain.
+Semua operasi tersebut diserahkan ke manusia.
+
+### Bekerja dengan Business Connection
+
+Ketika bot terhubung ke sebuah akun business, ia akan menerima sebuah update `business_connection`.
+Update tersebut juga akan diterima oleh bot ketika koneksinya tidak lagi tersambung atau telah terjadi perubahan sedemikian rupa terhadap koneksi tersebut.
+
+Contohnya, sebuah bot bisa jadi tidak bisa mengirim pesan ke chat yang dimaksud.
```
Comment (dzikrl) [inline site/docs/id/advanced/business.md:99]:
```suggestion
Contohnya, sebuah bot bisa jadi tidak dapat mengirim pesan ke chat yang dimaksud.
```

File: site/docs/uk/advanced/business.md (hunk 8 above)
Comment (deptyped) [reply site/docs/uk/advanced/business.md:79]:
```suggestion
## Просунуті можливості
```

**Review by deptyped — APPROVED** [review]

**Review by dzikrl — APPROVED** [review]

**Review by LWJerri — APPROVED** [review]
:shipit:

**Review by roj1512 — APPROVED** [review]

**Pull request merged**